docs/build/reference/integritycheck-require-signature-check.md

```diff
@@ -25,7 +25,11 @@ Microsoft has new signing guidance for DLL and executable files linked by using
 
 1. Select the **Configuration Properties** > **Linker** > **Command Line** property page.
 
-1. In **Additional Options**, enter *`/INTEGRITYCHECK`* or *`/INTEGRITYCHECK:NO`*. Choose **OK** to save your changes.
+1. To build an image which requires digital signature verification to be loaded, add *`/INTEGRITYCHECK`* to the **Additional Options** command line.
+
+1. To disable the feature, remove the *`/INTEGRITYCHECK`* option.
```

Comment:
This might be slightly clearer if we say "This feature is off unless turned on with `/INTEGRITYCHECK`."

Reply:
I updated this. If you have something you'd like better, let me know. In the meantime I'll get your changes through.